modules/contact-form/grunion-contact-form.php

```diff
@@ -844,16 +844,43 @@ public function personal_data_exporter( $email, $page = 1 ) {
 	 */
 	public function personal_data_eraser( $email, $page = 1 ) {
 		$per_page = 250;
-		$removed  = 0;
-		$retained = 0;
+		$removed  = false;
+		$retained = false;
 		$messages = array();
 		$post_ids = $this->personal_data_post_ids_by_email( $email, $per_page, $page );
 
 		foreach ( $post_ids as $post_id ) {
+			/**
+			 * Filters whether to erase the Feedback post.
+			 *
+			 * @since
```

Comment:
This can be set to `6.3.0` I believe. It's the next major release in the list of milestones.